### PR TITLE
ADD Tests for the Mongodb generic find

### PR DESCRIPTION
**Pull request by dmoranj**
Add a generic find function for the device registries, to allow users to find by any attribute.

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1 +1,2 @@
 - Allow listDevices() to use a single param (#180).
+- Implement generic find function for the device registries.
```
Comment (gtorodelvalle) [inline CHANGES_NEXT_RELEASE]: Final blank line appreciated... ;)

File: test/unit/mongodb-registry-test.js [hunk 2]
```diff
@@ -291,4 +291,52 @@ describe('MongoDB Device Registry', function() {
             });
         });
     });
+
+    describe('When a the registry is queried for a device using an arbitrary attribute', function() {
```
Comment (gtorodelvalle) [inline test/unit/mongodb-registry-test.js]: Minor typo: When "a the" registry...

**Comment by gtorodelvalle** [comment]
Probably some documentation would be great for the new functions but I leave it totally to you... :)
Apart from that and for 2 minor comments, LGTM from my side... ;)

**Comment by XavierVal** [comment]
LGTM

**Pull request merged**